aries-site/src/data/structures/templates.js

```diff
@@ -261,9 +261,12 @@ export const templates = [
   },
   {
     name: 'Matrix Navigation',
-    description: `Allows user to move back through previous
-    screens chronologically as you would on a browser
-    hierarchically within an application or website.`,
+    description: `Lets the user determine their own path by
```

Comment:
This should be divided up into a short into sentence to use on the card. The rest should be added to the mdx file.

This will require some slight re-wording to flow properly, but should be minor.

There is also a misspelling of Martix instead of Matrix.

![Screen Shot 2022-01-13 at 10 58 09 AM](https://user-images.githubusercontent.com/1756948/149384239-dc9e7d91-efae-48c6-b1f5-b9b03928ae7a.png)

Reply:
Thanks I copied this from figma so Ill let Vicky know of the typo as well